wp-content/themes/core/assets/pcss/global/stacking-order.pcss

```diff
@@ -0,0 +1,14 @@
+:root {
+	--tribe-stacking-order: 0;
+}
+
+/**
+ * need to target .is-not-stacked-on-mobile to make sure
+ * stacking order actually makes sense
+ */
+.wp-block-columns:not(.is-not-stacked-on-mobile) .wp-block-column.tribe-has-stacking-order {
```

Comment:
I love that we aren't neutering the `.is-not-stacked-on-mobile` option on the columns block. 🌮